Sasquatch/Sasquatch/ViewControllers/MSMainViewController.swift

```diff
@@ -170,10 +170,10 @@ class MSMainViewController: UITableViewController, AppCenterProtocol {
   @IBAction func logFilterSwitchChanged(_ sender: UISwitch) {
     #if ACTIVE_COMPILATION_CONDITION_PUPPET
     if !eventFilterStarted {
-      MSACAppCenter.startService(MSEventFilter.self)
+      AppCenter.startService(MSEventFilter.self)
```

Comment:
MSEventFilter?

Reply:
This is the class of our test app, not SDK